components/layout/footer.tsx

```
@@ -3,6 +3,13 @@
 import React from "react";
 import Link from "next/link";
 import { Github, Twitter, Linkedin, Mail } from "lucide-react";
+import { useDeviceCategory } from "@/hooks/use-breakpoint";
+import {
+  ResponsiveContainer,
+  ResponsiveGrid,
+  ResponsiveStack,
+  Show,
```

ESLint

[error] 'Show' is defined but never used.

(@typescript-eslint/no-unused-vars)